benchmarks/TFLite/linux-riscv.cmake

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+#                                                                              #
+# Default benchmark configurations                                             #
+#                                                                              #
+# Each suite benchmarks a list of modules with configurations specifying a     #
+# target architecture and runtime characteristics (e.g. threads/cores). These  #
+# benchmarks only configure IREE translation and runtime flags for the target  #
+# architecture and do *not* include any non-default flags. No non-default      #
+# flags should be added here.                                                  #
+#                                                                              #
+################################################################################
+
+set(LINUX_RV64_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv64"
+  "--iree-llvm-target-cpu=generic-rv64"
+  "--iree-llvm-target-abi=lp64d"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+d,+v"
+  "--riscv-v-vector-bits-min=512"
+  "--riscv-v-fixed-length-vector-lmul-max=8"
+)
+
+set(LINUX_RV32_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
```

Comment:
```suggestion
  "--iree-input-type=tosa"
  "--iree-llvm-debug-symbols=false"
```

Reply:
We want to keep the debug symbols in general so it is easy to triage the benchmark trace when we see the regression.

Therefore I think the proper way is to modify `iree_benchmark_suite` rule to generate two modules, one with debug symbols for benchmark run and one with `--iree-llvm-debug-symbols=false` for size measurement. We can do this in the follow-up changes.

Reply:
#9250 is the follow-up change for this.